consensus/broadcast/metaChainMessenger_test.go

```diff
@@ -303,3 +306,96 @@ func TestMetaChainMessenger_BroadcastBlockDataLeader(t *testing.T) {
 		assert.Equal(t, len(transactions), numBroadcast)
 	})
 }
+
+func TestMetaChainMessenger_Close(t *testing.T) {
+	args := createDefaultMetaChainArgs()
+	varModified := false
+	delayedBroadcaster := &mock.DelayedBroadcasterMock{
+		CloseCalled: func() {
+			varModified = true
+		},
+	}
+	args.DelayedBroadcaster = delayedBroadcaster
+	mcm, _ := broadcast.NewMetaChainMessenger(args)
+	if mcm == nil {
```

Comment:
I would do a require.NotNil(t, mcm) instead of return if mcm == nil, then the test would fail if mcm is nil, otherwise it will report success and that would be incorrect.